src/main/kotlin/us/spaceclouds42/ekho/Ekho.kt

```diff
@@ -46,13 +46,25 @@ class EkhoBuilder(base: LiteralText, method: EkhoBuilder.() -> Unit) {
         }
     }
 
+    fun text (text: MutableText, inheritStyle: Boolean = true, method: EkhoBuilder.() -> Unit = { }) {
```

Comment:
I like the idea, but currently it's not the best ux, with an extra layer of nesting. Using `operator fun`, this can be made even simpler:
```suggestion
operator fun Text.invoke(inheritStyle: Boolean = true, method: EkhoBuilder.() -> Unit = { }) {
```